### ISSUE TITLE
Adding of Tags/Releases and perhaps a maven 

### ISSUE THREAD

**Issue by ropetry** [Evolveum/midpoint-client-java#9]
Hello, would it be possible to add Tags and Releasae Version Numbers to the midpoint-cient-java project.. This would make it a lot easier for special / compatible versions 8-)..

Asso could we publish it to maven central so anyone can use it 8-)..
Thanks
bye
roman

**Comment by semancik** [comment]
Yes, we will. But this code was not officially released yet. It is still work in progress.
